migrations/20230627120414-create-look-up-table.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const fuzzySearchTableName = 'look_up';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable(fuzzySearchTableName, {
```

Comment:
I will relate `file.uuid` to `look_up.item_id` here so we are able to keep the table clean with `ON_DELETE=CASCADE` from files / folders via FK

Reply:
but item_id cannot reference files.uuid and folder.uuid at the same time. Do I create a look_up.file_id and look_up.folder_id to have the references?